visualization/FairnessDashboard/src/FairnessWizard.tsx

```diff
@@ -252,7 +252,7 @@ export class FairnessWizard extends React.PureComponent<IFairnessProps, IWizardS
                 parityMetrics: accuracyMetrics,
                 selectedParityKey: accuracyMetrics[0].key,
                 dashboardContext: FairnessWizard.buildPrecomputedFairnessContext(props),
-                activeTabKey: accuracyTabKey,
+                activeTabKey: featureBinTabKey,
```

Comment:
Is this the reason that the dashboard in Mark's part of the PoC always opens to the 'second' page?

Reply:
yes. This should fix it to open to the feature page